modules/Bio/EnsEMBL/DBSQL/TranscriptAdaptor.pm

```diff
@@ -250,6 +250,43 @@ sub fetch_all_versions_by_stable_id {
 }
 
 
+=head2 fetch_by_rnaproduct_id
+
+  Arg [1]    : Int $id
+               The internal identifier of the rnaproduct whose transcript
```

Comment:
rnaproduct => RNA product
or should it be $rnaproduct or RNAProduct as used in previous pods?